mne/commands/mne_browse_raw.py

```diff
@@ -93,6 +96,10 @@ def run():
         events = mne.read_events(eve_in)
     else:
         events = None
+
+    if filterchpi:
+        raw = mne.chpi.filter_chpi(raw)
```

Comment:
is

```python
if filterchpi and preload:
    raw = mne.chpi.filter_chpi(raw)
```

good enough, or should the `--filterchpi` flag also set `--preload` to True ?